src/pages/rental-details/rental-details.html

```diff
@@ -11,14 +11,14 @@
 
       <ion-item>
 				<ion-label floating color="primary">Start Date</ion-label>
-				<ion-input [(ngModel)]="details.startDate" name="startDate" type="date" #date="ngModel" required>
-				</ion-input>
+        <ion-datetime displayFormat="DDD D MMM, YYYY" pickerFormat="DD MMM YYYY" [(ngModel)]="details.startDate"  name="startDate" #date="ngModel" required>
+        </ion-datetime>
 			</ion-item>
 
       <ion-item>
 				<ion-label floating color="primary">Return Date</ion-label>
-				<ion-input [(ngModel)]="details.endDate" name="endDate" type="date" #date="ngModel" required>
-				</ion-input>
+        <ion-datetime displayFormat="DDD D MMM, YYYY" pickerFormat="DD MMM YYYY" [(ngModel)]="details.endDate"  name="endDate" #date="ngModel" required>
```

Comment:
It could be helpful to add a comment with examples of what this date format looks like, because I know I struggle with interpreting the `DDD D MMM` type formats. I also understand if you are not comfortable adding HTML comments, though, so I leave the decision up to you.